### ISSUE TITLE
Push to Live error

### ISSUE THREAD

**Issue by Norric1Admin** [ONSdigital/sdg-data#52]
https://github.com/ONSdigital/sdg-data/pull/51

Hello guys,
I made the first push (sdg-indicators) to update the metadata on the staging site with no problems, but the master <- develop PTL failed. I'm aware there's a new process since the split - can you advise please?
Thanks,
Chris

**Comment by dougmet** [comment]
Assuming it's this build https://travis-ci.org/ONSdigital/sdg-data/builds/394089099 it's saying that nothing changed. Which shouldn't cause and error but for some reason git gives a non-zero exit code for "no changes".

A work around could be to publish the build ID in the site README or something, which would guarantee something always changes.

**Comment by Norric1Admin** [comment]
Hi @dougmet, thanks for your response.
There are several changes, which have been updated on Staging but not the Live site (e.g. compare 1.5.4). How do I push the changes to .csv and .md files in 'data' and 'metadata' from 'sdg-data' to Live?

**Comment by Norric1Admin** [comment]
Hi @shaneporter, are you able to suggest a way to push our data and metadata to the live site? Any advice would be appreciated (note: I'm cloning the tool today so the answer might become apparent).

Thanks,
Chris

**Comment by Norric1Admin** [comment]
Solved a few of the problem (merge conflict with master) and the deployment was successful. The ‘master’ files are now up-to-date with ‘develop’. I can see the updated data/metadata in GitHub.

However, although everything appears to have been successful, the production site still hasn’t updated. I noticed that the Travis output is headed with the alert “Skipping a deployment with the script provider because this branch is not permitted: develop”. Unsure what to make of this.
